pkg/gardenlet/apis/config/v1alpha1/types.go

```diff
@@ -515,23 +515,23 @@ type ETCDController struct {
 	// Workers specify number of worker threads in ETCD controller
 	// Defaults to 3
```

Comment:
Not strictly related to this change but I think there is something wrong with the defaults:

https://github.com/gardener/gardener/blob/c52af1b21cc9a7340c1ca2574f107a2524a5c94c/pkg/operation/botanist/component/etcd/bootstrap.go#L330-L332.

I don't see any explicit defaulting in g/g for this field. There is only defaulting to `50` when ETCDController is non-nil and `Workers` is nil. Does the default `3` actually comes from etcd-druid? If yes, doesn't it make sense to update the above handling to use `3` instead of 50`?